app/serializers/route_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class RouteSerializer < ActiveModel::Serializer
+  attributes :id
+
+  belongs_to :user
```

Comment:
i think the user is not necessary because remember that the user will be saved in the mobile cache right after the user logs in. Please remove this.

Reply:
should the length attribute be added?